packages/styles/scss/components/footer/_footer-nav.scss

```diff
@@ -45,11 +45,12 @@
 
         display: block;
         column-count: 2;
-        column-gap: 0;
+        column-gap: 32px;
```

Comment:
maybe we could use the `$spacing-07` carbon token here https://www.carbondesignsystem.com/guidelines/spacing

Reply:
Maybe better would be `$carbon--grid-gutter` if we do have to go this route.

Reply:
I've applied this change on the latest commit